uMenu/romfs/fr.json

```diff
@@ -0,0 +1,132 @@
+    "control_minus": "Basculer vers le menu principal",
+    "suspended_app": "Application suspendu",
+    "suspended_close": "Voudriez-vous fermer ce titre ? Toutes les données non enregistrées seront perdues.",
+    "hb_launch": "Lancement du homebrew",
+    "hb_launch_conf": "Comment souhaiteriez-vous lancer cet homebrew ?",
+    "hb_applet": "Mode simple",
+    "hb_app": "Mode full",
+    "user_settings": "Paramètres utilisateur",
+    "user_selected": "Utilisateur sélectionné",
+    "user_option": "Que voudriez-vous faire avec cet utilisateur ?",
+    "user_pass_reg": "Créer un mot de passe",
+    "user_pass_ch": "Gérer le mot de passe",
+    "user_view_page": "Afficher la page utilisateur",
+    "user_logoff": "Se déconnecter",
+    "user_pass_ch_option": "Que voudriez-vous faire avec le mot de passe de l'utilisateur ?",
+    "user_pass_change": "Changer",
+    "user_pass_remove": "Supprimer",
+    "swkbd_user_pass_guide": "Entrer le mot de passe",
+    "swkbd_user_new_pass_guide": "Entrer le nouveau mot de passe",
+    "user_pass_change_conf": "Voulez-vous changé le mot de passe ?",
+    "user_pass_change_ok": "Le mot de passe a été modifié avec succès.",
+    "user_pass_change_error": "Une erreur s'est produite lors de la tentative de modification du mot de passe",
+    "user_pass_remove_full": "Supprimer le mot de passe",
+    "user_pass_remove_conf": "Souhaitez-vous vraiment supprimer le mot de passe de l'utilisateur ?",
+    "user_pass_remove_ok": "Le mot de passe a été supprimé avec succès.",
+    "user_pass_remove_error": "Une erreur s'est produite lors de la tentative de suppresion du mot de passe",
+    "user_pass_reg_conf": "Souhaitez-vous définir un mot de passe pour cet utilisateur ?",
+    "user_pass_reg_ok": "Le mot de passe a été configuré avec succès.",
+    "user_pass_reg_error": "Une erreur s'est produite lors de la tentative de configuration du mot de passe",
+    "user_logoff_app_suspended": "Il y a un titre ouvert. Souhaitez-vous vraiment le fermer et vous déconnecter ?",
+    "swkbd_webpage_guide": "Entrez l'URL de la page Web",
+    "set_unknown_value": "Inconnu",
+    "set_true_value": "Activer",
+    "set_false_value": "Désactiver",
+    "set_info_text": "Parcourir et/ou modifier les paramètres du système et d'uLaunch",
+    "set_console_nickname": "Pseudonyme de la console",
+    "set_console_timezone": "Emplacement du fuseau horaire de la console",
+    "set_viewer_enabled": "Connexion USB sur la visionneuse d'écran du PC (uViewer) activée",
+    "set_flog_enabled": "Prise de contrôle 'flog' en mode application Homebrew activée",
+    "set_wifi_none": "aucun (pas de connexion WiFi)",
+    "set_wifi_name": "Réseau WiFi connecté",
+    "set_console_lang": "Langue de la Console",
+    "set_console_info_upload": "Téléchargement des informations de la console activé",
```

Comment:
Originally, it is "upload". "Téléchargement" is for "download". Téléversement is the official translation for "upload", but you can also use "envoi"